testsuite/features/step_definitions/retail_steps.rb

```diff
@@ -9,19 +9,19 @@
   os_family = get_target('proxy').os_family
 
   # Distribution
-  repos = 'os_pool_repo os_update_repo'
-  repos += ' testing_overlay_devel_repo' unless $build_validation || product_version_full.include?('-released')
-  log get_target('proxy').run("zypper mr --#{action} #{repos}")
+  repos = 'os_pool_repo os_update_repo '
+  repos += 'testing_overlay_devel_repo ' unless $build_validation || $is_containerized_server || product_version_full.include?('-released')
 
   # Server Applications, proxy product and modules, proxy devel
   if os_family =~ /^sles/ && os_version =~ /^15/
-    repos = 'proxy_module_pool_repo proxy_module_update_repo ' \
-            'proxy_product_pool_repo proxy_product_update_repo ' \
-            'module_server_applications_pool_repo module_server_applications_update_repo'
-    repos += ' proxy_devel_releasenotes_repo proxy_devel_repo ' unless $build_validation || product_version_full.include?('-released')
+    repos += 'proxy_module_pool_repo proxy_module_update_repo ' \
+             'proxy_product_pool_repo proxy_product_update_repo ' \
+             'module_server_applications_pool_repo module_server_applications_update_repo '
+    repos += 'proxy_devel_releasenotes_repo proxy_devel_repo ' unless $build_validation || product_version_full.include?('-released')
   elsif os_family =~ /^opensuse/
-    repos = 'proxy_pool_repo'
+    repos += 'proxy_pool_repo ' unless $is_containerized_server
   end
+
   log get_target('proxy').run("zypper mr --#{action} #{repos}")
```

Comment:
I know this line is out of scope of this PR, but... that can be easy improved using `verbose: true` instead of using `log` of the returned value from run method.

Reply:
Yes, that was already so.

But yeah it does not cost much, i'll do as you say.

Reply:
Done together with the change to `run_local()`